frontend/components/Layout.tsx

```diff
@@ -1,11 +1,11 @@
-import Sidebar from './Sidebar';
+import MenuBar from './Menubar';
 import styled from 'styled-components';
 import { ReactElement } from 'react';
 
 const PageContainer = styled.div`
   width: 100vw;
```

Comment:
maybe make this 100% ?

Getting some weird behavior where when I extend my screen width, the background doesn't fill in

<img width="782" alt="Screen Shot 2022-02-15 at 11 52 11 PM" src="https://user-images.githubusercontent.com/17476012/154199241-7430b0b8-5015-411c-bf34-3f575a762be0.png">